gometalinter.json

```diff
@@ -2,12 +2,16 @@
   "Vendor": true,
   "Deadline": "2m",
   "Sort": ["linter", "severity", "path", "line"],
-  "Exclude": [
+  "Skip": [
       "cli/compose/schema/bindata.go",
       "cli/command/stack/kubernetes/api/openapi",
       "cli/command/stack/kubernetes/api/client",
       ".*generated.*",
-      "parameter .* always receives"
+      "vendor"
+  ],
+  "Exclude": [
+    "parameter .* always receives",
+    "_esc(Dir|FS|FSString|FSMustString) is unused"
```

Comment:
the commit message does not explain where this line is coming from

Reply:
Ahm, yes. Those are from the `cli/compose/schema/bindata.go` file. Looks like some linters don't get passed the `Skip` configuration correctly, and still invalidate.

We really need to get rid of gometalinter in this repository; there's a WIP pull request https://github.com/docker/cli/pull/1797, but I need to find some time to carry it.